src/style.css

```diff
@@ -0,0 +1,39 @@
+* {
+  box-sizing: border-box;
+}
+
+body,
+div {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+  padding-top: 2px;
+  padding-left: 2px;
```

Comment:
No need to write padding left and top for this task

Reply:
Not resolved.